### PR TITLE
Bilbo: Version 1.100 added

### PR DESCRIPTION
**Pull request by vv-monsalve**
fb05856: [gftools-packager] Bilbo: Version 1.100 added

* Bilbo Version 1.100 taken from the upstream repo https://github.com/googlefonts/bilbo at commit https://github.com/googlefonts/bilbo/commit/1a25794518fbf060b62a97dbf2effd1bb32da158.

 2824c51: [gftools-packager] ofl/bilbo remove METADATA "source".  google/fonts#2587

### CODE REVIEW

**Comment by vv-monsalve** [comment]
Part of Batch 1 of TypeSetIt projects

> 🔥 FAIL: Are there caret positions declared for every ligature?
> 🔥 FAIL: Is there kerning info for non-ligated sequences?

Related to: https://github.com/googlefonts/fontbakery/issues/3131

**Comment by m4rc1e** [comment]
Thanks!

File: ofl/bilbo/OFL.txt [hunk 1]
```diff
@@ -1,94 +1,94 @@
...
-MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT
-OF COPYRIGHT, PATENT, TRADEMARK, OR OTHER RIGHT. IN NO EVENT SHALL THE
-COPYRIGHT HOLDER BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
-INCLUDING ANY GENERAL, SPECIAL, INDIRECT, INCIDENTAL, OR CONSEQUENTIAL
-DAMAGES, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
-FROM, OUT OF THE USE OR INABILITY TO USE THE FONT SOFTWARE OR FROM
-OTHER DEALINGS IN THE FONT SOFTWARE.
+Copyright 2011 The Bilbo Project Authors (https://github.com/googlefonts/bilbo)
+
+
```
Comment (m4rc1e) [inline ofl/bilbo/OFL.txt:3]: Should just be a single line.

**Comment by vv-monsalve** [comment]
Updated

## Bilbo: Version 1.100 added

---
 d6f7752: [gftools-packager] Bilbo: Version 1.100 added

* Bilbo Version 1.100 taken from the upstream repo https://github.com/googlefonts/bilbo at commit https://github.com/googlefonts/bilbo/commit/40148a3e58ba010aa87c4ebf09f357ffb23475ff.

 4348681: [gftools-packager] ofl/bilbo remove METADATA "source".  google/fonts#2587

File: ofl/bilbo/OFL.txt (hunk 1 above)
Comment (vv-monsalve) [reply ofl/bilbo/OFL.txt]: Done.

**Pull request merged**